content/guides/hot-module-replacement.md

```diff
@@ -213,7 +213,7 @@ __webpack.config.js__
       new webpack.HotModuleReplacementPlugin()
     ],
     output: {
-      filename: 'bundle.js',
+      filename: '[name].build.js',
```

Comment:
I think `[name].bundle.js` is more commonly used than `[name].build.js` but I'm not sure if this is what is mentioned in the other guides. @TheDutchCoder do you know?

Personally I would prefer `[name].bundle.js` but I don't feel too strongly about it.